src/TLuaInterpreter.cpp

```diff
@@ -12008,13 +12113,15 @@ QString TLuaInterpreter::get_lua_string(const QString& stringName)
     return QString(lua_tostring(L, 1));
 }
 
-
 // check for <whitespace><no_valid_representation> as output
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#handleWindowResizeEvent -- not #noop - compare initLuaGlobals()
```

Comment:
This is an internal function

Reply:
Follow the link. It says: "(depreciated) This function is depreciated and should not be used; it's only documented here for historical reference - use the sysWindowResizeEvent event instead.
The standard implementation of this function does nothing. However, this function gets called whenever the main window is being manually resized. You can overwrite this function in your own scripts to ..."
So by internal you mean..?

Reply:
That is for `handleWindowResizeEvent`, but you put this above `noop`?

Reply:
Indeed. This was no accident. Some functions are published with a different name than defined internally. Compare like the comment indicates: 
`lua_register(pGlobalLua, "handleWindowResizeEvent", TLuaInterpreter::noop);`
Source: https://github.com/Kebap/Mudlet/blob/7bc88372dae822937a9ee161bc90241a9be5d3a6/src/TLuaInterpreter.cpp#L12483